prosody-plugins/mod_auth_token.lua

```diff
@@ -1,10 +1,17 @@
 -- Token authentication
 -- Copyright (C) 2015 Atlassian
 
+local basexx = require 'basexx'
+local have_async, async = pcall(require, "util.async");
+local formdecode = require "util.http".formdecode;
 local generate_uuid = require "util.uuid".generate;
+local http = require "net.http";
+local json = require 'cjson'
+json.encode_empty_table('array')
```

Comment:
No idea; probably copy/pasted from an example and wound up with some leftovers Will fix; thanks.